app/main.py

```diff
@@ -2,0 +1,17 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    mask_count = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(visitor=friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            mask_count += 1
+    if mask_count > 0:
+        return f"Friends should buy {mask_count} masks"
```

Comment:
Code Style Issue: Similar to the previous comment, it would be better to include the names of friends who are not wearing masks in the error message.